pkg/gadgets/snapshot/socket/tracer/tracer.go

```diff
@@ -193,3 +200,66 @@ func RunCollector(pid uint32, podname, namespace, node string, proto socketcolle
 
 	return sockets, nil
 }
+
+// ---
+
+type Tracer struct {
```

Comment:
This gadget is not sorting anymore because it is not implementing `SortByDefault`. BTW, the default sorting order it has before was:

https://github.com/inspektor-gadget/inspektor-gadget/blob/cb35dc8877b65a0549bfb0512b5012de3c8a303a/cmd/common/snapshot/socket.go#L70

Also, the process gadget is missing this feature.

Reply:
Ah, nice catch! I didn't realize we had this before.